content/en/quickstart/_index.md

```diff
@@ -46,11 +46,13 @@ This will install the Istio demo profile with ["Istio core" "Istiod" "Ingress ga
 ✔ Installation complete
 ```
 
-After the previous step is completed - the validation can be done by confirming GetMesh and Istio version installed:
+<h3>Verify it's Worked!</h3>
 
-```sh
-getmesh version
-```
+After the previous step is completed, you can use GetMesh to verify that the installation has succeeeded.
+
+**Check the Install Version**
```

Comment:
maybe: "Check the GetMesh Version" or "Installed Version"

Reply:
ack

Reply:
it's fixed in the updated file